src/services/dashboards/modules/DashboardRefreshDropdown.vue

```diff
@@ -33,14 +31,15 @@ import type { MenuItem } from '@spaceone/design-system/dist/src/inputs/context-m
 
 import { i18n } from '@/translations';
 
+import { useProxyValue } from '@/common/composables/proxy-state';
+
 import type { RefreshInterval } from '@/services/dashboards/config';
-import { refreshInterval } from '@/services/dashboards/config';
+import { refreshInterval, refreshIntervalMap } from '@/services/dashboards/config';
```

Comment:
suggestion: Sorry that I missed this at previous PR.. 🙏 I suggest you to change this constant name to upper case. That is our convention. 
These object must be used in other files(such as current component) and must not be writable.
In this case, plz use upper case :)

nitpick: I suggest to change the name of the constant. `refreshInterval` is an array. But I'm confused because the name is singular. 🥲 

nitpick2: Also, The word 'interval' is taken as having the return value of the `setInterval()` function.
What about 'refresh interval options'?

Reply:
suggestion: Also, this is not related in this component, but an incorrectly used type declaration was found.
At `@/services/dashboards/config`, I found out the code below:

```typescript
export const refreshInterval = ['off', '15s', '30s', '1m', '5m', '10m', '30m', '1h'];

export type RefreshInterval = typeof refreshInterval[number];
```
In this case, the type of `RefreshInterval` is actually declared as `string`.
If you want to declare individual values of `refreshInterval` as the type of `RefreshInterval`, you must write `as 
const`.

```typescript
export const refreshInterval = ['off', '15s', '30s', '1m', '5m', '10m', '30m', '1h'] as const;

export type RefreshInterval = typeof refreshInterval[number];
```